new/LIT7045Colic.xml

```diff
@@ -0,0 +1,84 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7045Colic" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Prayer against colic</title>
+                <title xml:lang="gez" xml:id="t2">ጸሎተ፡ ቍረፀት፡ ኄር፡ ኪሳ፡ ኪሳ፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Ṣalota qʷǝrḍat ḫer kisā kisā</title>
+                <title xml:lang="en" corresp="#t2">Prayer of the colic - Be good kisā kisā (?)</title>
```

Comment:
I woudl say: Prayer against the colic. Good kisā kisā ... ("Prayer of ..." sounds strange to me, perhaps you can say what is our preferable way of translating these titles).

Reply:
I wanted to express the contruct and make a difference in the translation in comparison to በእንተ፡ . However, I will change it.